@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -577,7 +578,8 @@ export class VisibleTileSet {
                     dataSource,
                     tileEntry.tileKey,
                     tileEntry.offset,
-                    frameNumber
+                    frameNumber,
+                    taskQueue
```

Comment:
see comment in MapView, make it a member, then it does not have to be passed around and can be accessed via this...